Engineering thesis/rozdzialy/Bibliografia.tex

```diff
@@ -0,0 +1,51 @@
+\begin{thebibliography}{1}
+\bibitem {Socket.IO} Socket.IO:
```

Comment:
po 1. 
masz dwa linki do socket.io i żaden nie prowadzi do oficjalnej dokumentacji. Usuń oba i wstaw link do socket io do oficjalnej strony.
po 2.
jak masz oficjalne źródła jak np. oficjalna dokumentacja, to powinieneś unikać dawać linków do wikipedii - wikipedia jest dobrym źródłem, ale równie dobrze wszystkie linki byłyby do wikipedii

Reply:
Zmieniłem na oficjalną stronę socket.io i dodałem githuba bo też korzystałem z niego przy pisaniu jakiś funkcji do obsługi